manuscript/manuscript.Rmd

```diff
@@ -423,7 +423,7 @@ Neural networks used in deep RL have proven remarkably effective in handling suc
 The rapidly expanding class of Model-free RL algorithms is particularly appealing given the ubiquitous presence of model uncertainty in ecological dynamics.
 Rarely do we know the underlying functional forms for ecological processes -- is population recruitment more Beverton-Holt or Ricker?
 Methods which must first assume something about the structure or functional form of a process before estimating the corresponding parameter can only ever be as good as those structural assumptions.
-Frequently available ecological data is insufficient to distinguish between possible alternative models [@devalpine], or the correct model may be non-identifiable with any amount of data.
+Frequently available ecological data is insufficient to distinguish between possible alternative models [@devalpine], or the correct model may be non-identifiable with any amount of data. <!-- ML: Having difficulty finding what the reference here should be. -->
```

Comment:
@cboettig If you have an idea of what the reference is lmk.

Reply:
there's a few things we might cite here, but really https://doi.org/10.1111/j.1461-0248.2011.01702.x is my favorite, because the idea that we cannot even convincingly detect something so simple and fundamental as density dependence in most population dynamics time series is a pretty arresting thought!